webcompat/static/js/lib/models/issue.js

```diff
@@ -231,6 +231,12 @@ issueList.IssueCollection = Backbone.Collection.extend({
       delete params.direction;
     }
 
+    // Issue 999, support domain name search
+    // Replace all site:DOMAIN_NAME to DOMAIN_NAME in:title
+    if (!!params.q.match(/site:([\w-\.]+(:\d+)?)/i)) {
+      params.q = params.q.replace(/site:([\w-\.]+(:\d+)?)/g, "$1 in:title");
```

Comment:
Can we create a local variable for the `/site:([\w-\.]+(:\d+)?)` regex, since we use it twice? That way if we need to tweak if, there's less of a chance or forgetting to change it in both places.